installation/licensing/set-up-your-license.md

```diff
@@ -1,35 +1,71 @@
 slug: set-up-your-license
 tags: maui,components,license,activate,download
 position: 1
 ---
 
-# Setting Up Your Telerik UI for .NET MAUI License Key
+# Installing Your Telerik UI for .NET MAUI License Key
 
 Starting with the Q1 2025 release, the UI components from the Telerik UI for .NET MAUI library require activation through a license key (trial or commercial). This article describes how to download your personal license key and use it to activate the Telerik UI for .NET MAUI components.
 
 An invalid license results in [errors and warnings]({%slug license-errors-warnings%}) during build and run-time indicators such as watermarks and banners.
 
-## Downloading the License Key
-
 To download a license key for Telerik UI for .NET MAUI, you must have either a developer license or a trial license. If you are new to Telerik UI for .NET MAUI, sign up for a [free trial](https://www.telerik.com/try/ui-for-maui) first, and then follow the steps below.
 
+Depending on your development environment and preferences, you can install your license key in either of the following ways:
+
+* [Automatic Installation for projects with NuGet references](#automatic-license-key-installation)&mdash;Suitable for developers using Telerik productivity tools like the Telerik extensions for Visual Studio and the Progress Control Panel
+* [Manual installation for projects with NuGet references](#manual-license-key-installation)&mdash;Suitable for trial users and developers who prefer to manage their projects and Telerik product versions manually.
+* [Manual installation for projects using assembly references (no NuGet packages)](#adding-a-license-key-in-projects-without-nuget-references)&mdash;Suitable for developers who cannot use NuGet references in their projects and use the Telerik DLLs instead.
+
+## Automatic License Key Installation
+
+Telerik provides tools that automatically provision your license key. These tools include the [Progress Control Panel]({%slug control-panel%}), the [Visual Studio Extensions]({%slug vs-integration-overview%}) and [Visual Studio Code extensions]({%slug getting-started-vs-code-integration-overview%}).
+
+### Installing a License Key with the VS Extensions
+
+To install your license key by using the [Telerik UI for .NET MAUI Visual Studio extensions]({%slug vs-integration-overview%}):
+
+1. Open Visual Studio.
+1. Go to **Extensions** > **Telerik** **Licensing** > **Download Key**.
+
+    ![.NET MAUI VS Extension License Key](./images/vsx-download-license-key-file.png)
+
+### Installing a License Key with the VS Code Extensions
+
+To install your license key by using the [Telerik UI for .NET MAUI Visual Studio Code extensions]({%slug getting-started-vs-code-integration-overview%}):
+
+1. Open Visual Studio Code
+1. Follow the instructions described in the [Start the Wizard article]({%slug getting-started-vs-code-integration-new-project%}#start-the-wizard) for opening the Telerik Extensions menu.
```

Comment:
The instructions in this section should be complete (without referring to other articles). This is vital, as the procedure is critical for using the product. Sending the reader to another article and then asking them to come back here makes the procedure look very complex and could lead to significant frustration.